include/openssl/bnerr.h

```diff
@@ -49,6 +49,7 @@ int ERR_load_BN_strings(void);
 #  define BN_F_BN_EXP                                      0
 #  define BN_F_BN_EXPAND_INTERNAL                          0
 #  define BN_F_BN_GENCB_NEW                                0
+#  define BN_F_BN_GENERATE_DSA_DETERMINISTIC_NONCE         0
```

Comment:
Don't do this.  Instead just use "0" inline directly in your ERR-put calls.
Because changes to generated files (you saw the "do not edit" comment, right?) will get clobbered by 'make update'